### ISSUE TITLE
Null check operator used on a null value

### ISSUE THREAD

**Issue by somtech123** [sbis04/video_trimmer#228]
E/MediaMetadataRetrieverJNI(28521): getFrameAtTime: videoFrame is a NULL pointer
I/flutter (28521): ERROR: Couldn't generate thumbnails: MissingPluginException(No implementation found for method data on channel plugins.justsoft.xyz/video_thumbnail)
Another exception was thrown: Null check operator used on a null value
E/MediaMetadataRetrieverJNI(28521): getFrameAtTime: videoFrame is a NULL pointer
I/flutter (28521): ERROR: Couldn't generate thumbnails: MissingPluginException(No implementation found for method data on channel plugins.justsoft.xyz/video_thumbnail)
Another exception was thrown: Null check operator used on a null value

**Comment by Kasun-thilina** [comment]
It was a bug from the thumbnail generation. I have added a few fixes for that in a separate repository. Check this one out,

  ```
video_trimmer:
    git:
      url: https://github.com/Kasun-thilina/video_trimmer
      ref: v3.0.5
```

**Comment by pkphone** [comment]
Any updates?

**Comment by sbis04** [comment]
Should be fixed in the latest version `4.0.0`!
